pkg/zeek/escape.go

```diff
@@ -23,26 +23,32 @@ func Escape(data []byte) string {
 	return string(buf)
 }
 
+func ParseEscape(data []byte) (byte, int) {
+	if len(data) >= 4 && data[1] == 'x' {
+		v1 := unhex(data[2])
+		v2 := unhex(data[3])
+		if v1 <= 0xf || v2 <= 0xf {
+			return v1<<4 | v2, 4
+		}
+	}
+	return data[1], 2
+}
+
```

Comment:
Nit: Put this after Unescape so it can be next to its buddy, unhex.

Reply:
+1